drivers/hmis/app/models/hmis/hud/enrollment.rb

```diff
@@ -246,7 +255,18 @@ def assign_unit(unit:, start_date:, user:)
   end
 
   def release_unit!(occupancy_end_date = Date.current, user:)
-    active_unit_occupancy&.occupancy_period&.update!(end_date: occupancy_end_date, user: user)
+    occupancy = active_unit_occupancy
+    if occupancy&.occupancy_period&.nil?
```

Comment:
welp
```
[3] pry(#<Hmis::Hud::Enrollment>)> occupancy.nil?
=> true
[4] pry(#<Hmis::Hud::Enrollment>)> occupancy&.occupancy_period&.nil?
=> nil
```

pushed a fix for this issue that came up in tests. if tests pass I will merge

Reply:
Ah shoot, sorry! Thanks for fixing! I thought CI was just failing due to rubocop/audit checks. I should have double checked for test failures 🤦 

 This should have been:
```suggestion
    if occupancy&.occupancy_period.nil?
```
Although IMO much better to use `if !occupancy&.occupancy_period` (which should now pass rubocop)

Reply:
ahh I see. its ok, I know we really need to get those green, I've done the same thing